langs/th.json

```diff
@@ -0,0 +1,189 @@
+{
+  "locale": "th",
+  "languages": {
+    "el": "\u0e01\u0e23\u0e35\u0e01",
```

Comment:
for consistency over all languages, this should rather have the utf8 characters instead of the escape codes. something along this line fixes this quickly:

```
fs.writeFileSync(
  './langs/th.json',
  JSON.stringify(
    JSON.parse(fs.readFileSync('./langs/th.json')),
    null,
    2
  )
);
```

for the same reason, pleas have the language in alphabetical order of the language codes.